src/__mocks__/response.ts

```diff
@@ -0,0 +1,86 @@
+// tslint:disable:no-any
```

Comment:
you should use [jest-mock-express](https://www.npmjs.com/package/jest-mock-express) instead